src/main/java/com/librato/disco/DiscoService.java

```diff
@@ -88,9 +88,25 @@ public void stop() throws Exception {
     }
 
     private void createNode() throws Exception {
-        framework.create()
-                .withMode(CreateMode.EPHEMERAL)
-                .forPath(node, payload);
+        int retries = 15;
+        long sleepMillis = 1000;
+        Exception exception = null;
+
+        while (retries > 0) {
+            try {
+                framework.create()
+                        .withMode(CreateMode.EPHEMERAL)
+                        .forPath(node, payload);
+                return;
+            } catch (Exception e) {
```

Comment:
I saw Eric's response in Slack about force deleting. If we don't do that, I'd at least ensure that the exception is because the node exists and not something else. For anything but node exists I think we'd want to bail immediately.